src/Button.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { space } from 'styled-system';
+import { space, fontSize, color } from 'styled-system';
```

Comment:
What's the benefit of exposing `fontSize` and `color` here? I might have missed where we are utilizing it, however it feels counter-intuitive to the variants we support for button. We can end up with inconsistent states where someone specifies a variant but then misuses the colors. Similarly with the size prop, we can end up with misused sizes and fontSizes.

Reply:
I agree with you here. I added `fontSize` because the spacing in shot cards wasn't working and ended up sticking to the system here so it should be removed. As for `color`, we'll have to set many more variants to allow for shot status buttons - using this as a workaround currently. Have we looked into using styled-system's `colorStyles` / `buttons` [style utilities](https://github.com/jxnblk/styled-system/blob/master/docs/api.md#variants) at all?

This is also team-discussion-worthy, but I still believe these types of cr-specific presentational components (shot status button, shot card, etc) should live with the rest of our Arbor components, potentially as patterns. I get that we're trying to enable anyone to use Arbor, but first and foremost it should serve as a tool that lets us move as quickly as possible, and splitting up the work feels counterintuitive with things slipping through the cracks - these should be independent enough that as soon as, say, our Button component is ready for production - it should be able to be used in projects that are currently in progress. There are plenty of generic design systems out there.

Reply:
I’m still concerned with adding color here and now in this PR. It’s very much unrelated to the checkbox and is assuming some details on how exactly we will implement the shot status component. We may use buttons directly or we may not. It feels like a decision that shouldn’t be made at this time.